src/diffusers/schedulers/scheduling_dpmsolver_multistep_flax.py

```diff
@@ -0,0 +1,563 @@
+# Copyright 2022 TSAIL Team and The HuggingFace Team. All rights reserved.
```

Comment:
The same proposed changes as in the PyTorch file

Reply:
fixed in https://github.com/huggingface/diffusers/commit/f54cf995f4106e835513073c75e38b26da3eea01